common-practices-tools/security/yubikey.md

```diff
@@ -0,0 +1,309 @@
+  - YubiKey (an easier option) is available when using [LastPass Premium](https://www.lastpass.com/yubico/) ($12/year)
+    - Select the YubiKey option.
+    - Insert the YubiKey device into a USB port on your computer.
+    - Focus your cursor on the "YubiKey #1" field.
+    - Press the button on the YubiKey device.
+    - A long string of dots should appear in the YubiKey #1 field.
+    - Change the "YubiKey Authentication" status to "Enabled"
+    - Set "Enabled" ==> "Yes"
+    - Set "Permit Offline Access" ==> "Disallow"
+    - Press the Update button
+    - Enter your LastPass master password and press Confirm.
+    - YubiKey is now enabled for your LastPass account.
+- If your phone supports NFC, you can touch the more recent Yubikeys against your phone to unlock on mobile.
+
+<!-- prettier-ignore-end -->
+
+### Google
+
+For each Google account you have:
+
+-   Visit <https://accounts.google.com/b/0/SmsAuthSettings#devices>
+-   Enable TFA, and complete the phone verification process (phone will act as backup TFA).
+-   Click on "Security Keys" and follow instructions to add YubiKey.
+-   Return to the main page and add a second phone and/or print backup codes.
+-   As long as you have a backup, you can also install the YubiKey Authenticator app, and configure your account to use that for the backup TFA instead of SMS/phone - this is the same as the Google Authenticator app, except that it stores the credentials on your YubiKey instead of the phone.
+-   If you have funky devices/apps that don't support TFA, you can set an application specific password using that tab. This includes sending E-mail from your personal Gmail account using your civicactions.com IMAP, for instance.
+
+### GitHub
+
+-   Visit <https://github.com/settings/two_factor_authentication/configure>
+-   Enable TFA, and complete the phone verification process (phone will act as backup TFA).
+-   Then you can "Register new device" in the "Security keys" section
+
+### AWS Root Account
+
+For each AWS account you have:
+
+-   Visit <https://console.aws.amazon.com/iam/home?region=us-east-1#security_credential>
+-   Under MFA, add a Virtual MFA device.
+-   Use YubiKey Authenticator app to scan the QR code, and enter the response code, then close and reopen the app and enter the second response code.
+
+### AWS IAM Account
+
+Up to 8 different MFA devices can be assigned, with a combination of Yubikeys, hardware TOTP tokens and authenticator apps.
+
+-   Visit <https://console.aws.amazon.com/iam/home?region=us-east-1#users>
+-   Choose your user name
+-   Select the "Security Credentials" tab
+-   Under Multi-factor Authentication(MFA), select "Assign MFA device"
+-   Enter a Device Name and Select "Security Key"
+-   Allow your browser access to your Yubikey, if prompted
+-   Enter your Yubikey's pin, if prompted
+-   Touch your Yubikey when instructed
+
+## YubiKey Neo U2F Setup
```

Comment:
We should not support Yubikeys before 5
Remove up to "Locking Your machine with Yubikey" once policy is updated...
Policy question for Marc and Owen: "CivicActions will replace a CivicActions' purchased Yubikey before version 5"

Reply:
Ok, I'm removing it.